src/jit/emitxarch.cpp

```diff
@@ -10718,6 +10753,7 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             sz  = emitSizeOfInsDsc(id);
             break;
         case IF_RWR_RRD_RRD_CNS:
+        case IF_RWR_RRD_RRD_RRD:
```

Comment:
Is it OK to treat `IF_RWR_RRD_RRD_CNS` and `IF_RWR_RRD_RRD_RRD` as same? Constant is a part of instruction encoding and does not need a register and my worry is that this may impact emitter at later stages. I have experienced this with `IF_RRW_CNS` and `IF_RRW_RRW_CNS`.

Reply:
I don't think we should treat these as same. There is other handling and/or encoding checks that this can potentially impact.

Reply:
> Is it OK to treat IF_RWR_RRD_RRD_CNS and IF_RWR_RRD_RRD_RRD as same? 

At least, for JITDump, we cannot.